README.md

```diff
@@ -933,3 +935,15 @@ The typical workflow involves making a synchronous API call to the POST endpoint
 ## How to use the API's
 
 The API's are documented at [https://adobedocs.github.io/photoshop-api-docs/#api-Sensei-ImageCutout](https://adobedocs.github.io/photoshop-api-docs/#api-Sensei-ImageCutout)
+
+# Lightroom APIs
+
+The Adobe Lightroom APIs allow you to make Lightroom like automated edits to JPG image files.
```

Comment:
@shayanc1985 
"The Adobe Lightroom APIs allow you to apply Lightroom-like, automated edits to JPG image files."

Reply:
@noahhoffman I realized that i need to remove JPG and make it just "image files" since we support multiple input formats.